### PR TITLE
CLOUDSTACK-9026: Modifying testpath for adding missing parameter

### PR DESCRIPTION
**Pull request by pritisarap12**
Adding service_offering creation in testpath_storage_migration.py testpath which is missing right now

### CODE REVIEW

**Comment by pritisarap12** [comment]
Testcase result:

Test migrate Volume (root and data disk) ... === TestName: test_01_migrate_root_and_data_disk_nonlive | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 1688.400s

OK

**Comment by sanju1010** [comment]
LGTM  based on test results!!

Test migrate Volume (root and data disk) ... === TestName: test_01_migrate_root_and_data_disk_nonlive | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 1778.920s

OK

File: test/integration/testpaths/testpath_storage_migration.py [hunk 1]
```diff
@@ -248,6 +248,11 @@ def setUpClass(cls):
                 DomainName=cls.account.domain
             )
             # Create Service offering
+            cls.service_offering = ServiceOffering.create(
+                cls.apiclient,
+                cls.testdata["service_offering"]
+            )
+
```
Comment (nitt10prashant) [inline test/integration/testpaths/testpath_storage_migration.py]: please add cls._cleanup.append(cls.service_offering)

Comment (pritisarap12) [inline test/integration/testpaths/testpath_storage_migration.py]: Added service_offering in cleanup.

**Comment by shwetaag** [comment]
did cod walk thought. Looks good to me .

**Pull request merged**